src/main/java/org/gridsuite/study/server/networkmodificationtree/dto/InsertMode.java

```diff
@@ -13,5 +13,6 @@
 public enum InsertMode {
     BEFORE,
     AFTER,
-    CHILD;
+    CHILD,
+    NEW_BRANCH;
```

Comment:
CHILD and NEW_BRANCH seems to have same signification ...
maybe remove CHILD ?

Reply:
> `Paste node in a new branch ` create the node above and not below the selected node

fixed

Reply:
done